packages/camera/camera/example/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Demonstrates how to use the camera plugin.
 publish_to: none
 
 environment:
-  sdk: ^3.4.0
+  sdk: ^3.6.0
```

Comment:
Why is the Dart SDK version being changed in all the pubspecs? What new Dart feature does this PR rely on?